test/e2e/docker_test.go

```diff
@@ -590,6 +590,22 @@ func TestDockerKubernetes127AirgappedRegistryMirrorAndCert(t *testing.T) {
 	runDockerAirgapConfigFlow(test)
 }
 
+func TestDockerKubernetes127AirgappedUpgradeFromLatestRegistryMirrorAndCert(t *testing.T) {
+	release := latestMinorRelease(t)
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewDocker(t),
+		framework.WithClusterFiller(api.WithExternalEtcdTopology(1)),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube127)),
+		framework.WithAuthenticatedRegistryMirror(constants.DockerProviderName),
+	)
+	runDockerAirgapUpgradeFromReleaseFlow(
+		test,
+		release,
+		v1alpha1.Kube127,
+	)
```

Comment:
what kind of upgrade is this running? it doesn't seem to pass any changes

Reply:
It's only upgrading from the latest minor release of EKS-A to the current branch. I could add a kubernetes version change or something, but I think this is also enough to test the bundle upgrades.

Reply:
ahhh got you no that works
if we are not passing spec changes, then let's remove the options in the runDockerAirgapUpgradeFromReleaseFlow method